Automattic/vip-go-mu-plugins
# Add upper and lower limits to JP sync queue settings

## Description

VIP sites need higher defaults for the `max_queue_size` and `max_queue_lag` Jetpack Sync settings - this adds two option filters to ensure that the settings are within a predefined range, while still allowing the exact setting to be tweaked on a per-site basis (compared to a `pre_option` filter).

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check a site's settings - `wp option get jetpack_sync_settings_max_queue_size`
1. Should be default (`1000`)
1. Check out PR.
1. Check a site's settings - `wp option get jetpack_sync_settings_max_queue_size`
1. Should now be the minimum size allowed - `10000`
